wagtail/wagtailadmin/views/account.py

```diff
@@ -126,6 +125,7 @@ def login(request):
 def logout(request):
     response = auth_views.logout(request, next_page='wagtailadmin_login')
 
+    messages.success(request, 'You have successfully logged out!')
```

Comment:
Thanks, Liam! The message should be a translatable string, e.g. https://github.com/takeflight/wagtail/blob/logout-message/wagtail/wagtailadmin/views/account.py#L59. I'm not sure we need the `!`.